src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,28 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector INJECTOR = Injector.getInstance("mate.jdbc");
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("mate.jdbc");
```
because it can change the internal state